CONTRIBUTING.md

```diff
@@ -23,6 +23,7 @@ We welcome all kinds of contributions, from improving customization, to extendin
 
 ### **Submit Pull Requests** 💻
 - Fork the repository and create a new branch for your changes.
+- Install [pre-commit](https://pre-commit.com/) to ensure the code is formatted and standardized correctly, by running `pip install pre-commit` and then `pre-commit install`.
```

Comment:
@Kostis-S-Z for a small change like this, don't hesitate to push directly to main as I don't think a review is needed.

Reply:
Sounds good, i ll keep it in this PR in this case, but keep it in mind for the future.